primefaces/src/main/java/org/primefaces/csp/CspPhaseListener.java

```diff
@@ -67,7 +70,15 @@ public void beforePhase(PhaseEvent event) {
         CspState state = PrimeFacesContext.getCspState(context);
 
         String policy = LangUtils.isBlank(customPolicy.get()) ? "script-src 'self'" : customPolicy.get();
```

Comment:
i wonder of this should also be in the `else` statemen as customPolicy should not be used in report-only mode?

Reply:
Ahhh wait a minute.  I think his original idea was correct.

So in report only mode it would generate...

```
script-src nonce-12344 report-uri \myreports
```

There is no way a user can generate `script-src nonce-12344 ` so the reports won't be reporting anything.   I think I have to put it back the way it was.